src/coreclr/src/jit/codegenarm64.cpp

```diff
@@ -4793,7 +4790,7 @@ void CodeGen::genStoreLclTypeSIMD12(GenTree* treeNode)
     regNumber operandReg = genConsumeReg(op1);
 
     // Need an addtional integer register to extract upper 4 bytes from data.
-    regNumber tmpReg = treeNode->GetSingleTempReg();
+    regNumber tmpReg = lclVar->GetSingleTempReg();
```

Comment:
Can you fix the typo in the line above? addtional -> additional

Reply:
Done.